P4D2_2017_Fall/vm/user-bootstrap.sh

```diff
@@ -126,7 +126,7 @@ cd .vim
 mkdir ftdetect
 mkdir syntax
 echo "au BufRead,BufNewFile *.p4      set filetype=p4" >> ftdetect/p4.vim
-echo "set bg=dark" >> /home/p4/.vimrc
+echo "set bg=dark" | sudo tee /home/p4/.vimrc
```

Comment:
`tee` will override vimrc file. You should use ` tee --append`

Reply:
Thank you so much.